tslib/api-session.ts

```diff
@@ -91,7 +91,7 @@ export function makeHandler (): (req: OurRequest, res: OurResponse, next: Functi
                     if (needComma) {
                         chunk += ',';
                     }
-                    chunk += '\'data\':[';
+                    chunk += '\"data\":[';
```

Comment:
Since the outer string is encapsulated in single quotes, `'`, you can remove escape slash (i.e., `\`).